### ISSUE TITLE
Package name (except the service name) should tell us the usage of the package

### ISSUE THREAD

**Issue by pamungkaski** [helloproclub/cassiopeia#3]
i think package Hall Of Fame should not be called Hall of fame but instead Model / Recorder or something that tell us that it is used to get data from DB

**Comment by satriahrh** [comment]
Yes, I do not have a name yet. It is taken from the section name on the mock up design. The name is still open for suggestion.

**Comment by pamungkaski** [comment]
Yep, one of the hardest thing in coding are naming things

**Comment by satriahrh** [comment]
Thanks @pamungkaski  for making this impossible on #4
